corehq/motech/repeaters/repeater_generators.py

```diff
@@ -319,8 +319,8 @@ def _set_constant_properties(case, config):
                 if key.startswith(constant_prefix):
                     property_name = key[len(constant_prefix):]
                     constant_properties.append((property_name, value))
-            whitelist = payload_doc.get_case_property(f'{case_type}_whitelist')
-            blacklist = payload_doc.get_case_property(f'{case_type}_blacklist')
+            whitelist = payload_doc.case_json.get(f'{case_type}_whitelist', '').split(' ')
```

Comment:
`''.split('')` will return `['']` which will be truthy, won't that be a problem?

Reply:
Yep, this was in fact a problem, and is fixed in the next commit